packages/provider/src/tasks/powCaptcha/powTasks.ts

```diff
@@ -51,8 +51,10 @@ export class PowCaptchaManager {
 		userAccount: string,
 		dappAccount: string,
 		origin: string,
+		powDifficulty?: number,
 	): Promise<PoWCaptcha> {
-		const difficulty = 4;
+		const difficulty = powDifficulty || 4;
```

Comment:
Set default as a variable somewhere that is easier to see

Reply:
Done